packages/core/src/test/shared/wizards/wizardTestUtils.ts

```diff
@@ -65,9 +65,7 @@ function failIf(cond: boolean, message?: string): void {
 export async function createWizardTester<T extends Partial<T>>(wizard: Wizard<T> | WizardForm<T>): Promise<Tester<T>> {
     if (wizard instanceof Wizard && wizard.init) {
         // Ensure that init() was called. Needed because createWizardTester() does not call run().
-        ;(wizard as any)._ready = true
```

Comment:
is this is needed can you make it part of a separate PR along with the other wizard changes

Reply:
You can just create a PR -> have that one merged first, merge/rebase this one and you're good to go